crates/ruff_python_ast/src/binding.rs

```diff
@@ -265,6 +265,26 @@ pub enum BindingKind<'a> {
     SubmoduleImportation(SubmoduleImportation<'a>),
 }
 
+impl<'a> BindingKind<'a> {
+    pub fn qualified_name(&self) -> Option<&'a str> {
+        match self {
+            BindingKind::Importation(importation) => Some(importation.full_name),
+            BindingKind::FromImportation(importation) => Some(&importation.full_name),
```

Comment:
I also considered changing `FromImportation` to store the normalized fields and create `full_name` on-demand:

```rust
#[derive(Clone, Debug)]
pub struct FromImportation<'a> {
    /// The name to which the member is bound.
    /// Given `from foo import bar`, `name` would be "bar".
    /// Given `from foo import bar as baz`, `name` would be "baz".
    pub name: &'a str,
    /// The level of the import. `None` or `Some(0)` indicate an absolute import.
    pub level: Option<usize>,
    /// The module being imported.
    pub module: Option<&'a str>,
    /// The member being imported.
    pub member: &'a str,
}
```